### PR TITLE
[ADD] whatsapp: basic description of the app

### PR DESCRIPTION
**Pull request by hapa-odoo**
task-3425722

### CODE REVIEW

File: content/applications/productivity/whatsapp.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+========
+WhatsApp
+========
+
+**WhatsApp** is a popular messaging app that enables users to send texts, make calls, and share
```
Comment (jva-odoo) [inline content/applications/productivity/whatsapp.rst:5]:
```suggestion
**WhatsApp** is a popular messaging app that enables users to send texts, and share
```
No calls allow for a business account

File: content/applications/productivity/whatsapp.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+media securely and conveniently. Businesses can use `WhatsApp Business
+<https://business.whatsapp.com>`_ to communicate with their customers, send documents and provide
```
Comment (jva-odoo) [inline content/applications/productivity/whatsapp.rst:7]: The link for this is https://developers.facebook.com/products/whatsapp/ 
The above link is for normal WhatsApp business

File: content/applications/productivity/whatsapp.rst (hunk 1 above)
Comment (jcs-odoo) [reply content/applications/productivity/whatsapp.rst:5]: Ah indeed, thanks, but I thought it'd be interesting to briefly describe the "normal" whatsapp in the first sentence. In the second paragraph, Idon't list calls as features we can do. What do you think?

File: content/applications/productivity/whatsapp.rst (hunk 2 above)
Comment (jcs-odoo) [reply content/applications/productivity/whatsapp.rst:7]: Thank you @jva-odoo  :)

File: content/applications/productivity/whatsapp.rst (hunk 1 above)
Comment (jva-odoo) [reply content/applications/productivity/whatsapp.rst:5]: No, it gives a wrong expectation to the customer.
because both are totally different.

**Comment by jva-odoo** [comment]
@odoo/rd-runbot Hello team,
add a code owner for the WhatsApp app

**Comment by jcs-odoo** [comment]
> @odoo/rd-runbot Hello team, add a code owner for the WhatsApp app

Hi @jva-odoo I'll do it with @AntoineVDV as we need to add a GitHub team as code owner.

**Comment by jcs-odoo** [comment]
@jva-odoo it's done

**Comment by jva-odoo** [comment]
> @jva-odoo it's done

Thank you

**Comment by jcs-odoo** [comment]
Moving this to https://github.com/odoo/documentation/pull/5713 so that I could create a related branch on odoo/enterprise (PR https://github.com/odoo/enterprise/pull/46785)

**Pull request closed without merging**